lambdas/contra-indicator-storage-core-stub/src/main/java/uk/gov/di/ipv/core/contraindicatorstoragecorestub/domain/GetCiRequest.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.di.ipv.core.contraindicatorstoragecorestub.domain;
+
+import lombok.Getter;
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+
+@ExcludeFromGeneratedCoverageReport
+@Getter
+@SuppressWarnings("java:S116") // Field names should comply with a naming convention
```

Comment:
We previously used the Jackson @JsonProperty annotation here to handle the snake-case var names. I guess the downside is introducing another gradle dependency but would avoid having to suppress the warning which might be nicer?


But not too fussed about this so happy for you to decide what seems better.

Reply:
Yeah, I was trying to not use Jackson. The auth team found a lot of speed up in their lambdas by switching from Jackson to Gson.
I think I've just found something that can do a similar thing to `@JsonProperty` though so I might try adding it in.

Reply:
Ah okay yeah thats fair enough, happy to leave out jackson then 👍